### DISCUSSION TITLE
Running cis tests against kubernetes > 1.16

### DISCUSSION THREAD

**Discussion by ratnopam** [aquasecurity/kube-bench#725] [Questions and Help] [answered]
In the `README.md` it's mentioned that latest cis version is 1.6 and it conforms to kubernetes 1.16. 

If a kubernetes cluster running 1.17, 1.18 or 1.19 has to be certified, then can I run cis benchmark tests (for cis 1.6) with kube-bench against that cluster? 

Can you please advise. Thanks.

**Comment by mozillazg** [comment, marked as answer]
@eratnch Yes, cis 1.6 is for kubernetes 1.16+
